Dockerfile

```diff
@@ -10,6 +10,10 @@ ARG BUILD_BRANCH=master
 RUN git clone --depth=1 --recursive --shallow-submodules -b $BUILD_BRANCH https://github.com/ONLYOFFICE/document-server-integration.git
 WORKDIR /document-server-integration/web/documentserver-example/nodejs/
 
+RUN sed -i "s/\"enableForgotten\"\: false\,/\"enableForgotten\"\: true\,/" config/default.json
+RUN sed -i "s/1073741824/107374182400/g" config/default.json
```

Comment:
I think it will be cleaner if you will replace not `123456789` to `987654321` without ability to understand what this lines do, but instead replace

```
s/super_important_value: 123456789/super_important_value: 987654321/g
```
or maybe even better - since we replace in json file - use `jq` instead of plain `sed` - what way we will exactly see that are we replacing

If we take line 13 it will be something like
```
jq '.enableForgotten = true' config/default.json
```

only thing that jq do not support inline edit of files, but utility sponge from `moreutils` seems support it:

So if I not mistaken final solution will be
```
jq '.enableForgotten = true' config/default.json | sponge config/default.json

```
But please double check it